src/project_generator.rs

```diff
@@ -49,6 +64,16 @@ impl Generator {
         let content = self.get_problem_content().await?;
 
         html::generate_markdown(self.project_title.clone(), &content, self.directory.clone())?;
+
+        println!(
+            "{}",
+            format!(
+                "🌟 Leetcode project was created in dir {} with language {}",
+                &self.directory.bold(),
+                &self.config.default_lang
+            ).green()
+        );
+
```

Comment:
I'm okay with this, but when creating a project for rust with `Command::new("cargo").arg("new")...` it already prints an output `Created binary (application) ...`

Reply:
Also, I think it's better to use log::info!() as some users might not want an output at all, and with `env_logger` the user would have an option to disable any messages.